documentation/src/main/asciidoc/index.asciidoc

```diff
@@ -2,58 +2,56 @@
 Written and maintained by the {brandname} community
 :icons: font
 
-Get answers and find help in the {brandname} {infinispanversion} titles.
+Get answers and find help in the {brandname} {infinispanversion} doc set.
 
-Getting Started::
+Embedding Infinispan::
```

Comment:
There are some "Infinispan" words in the document. should they be changed to "{brandName}"?

Reply:
the index will always be upstream only so hardcoding doesn't matter. better to be consistent though. I'll replace with the brandname variable.